packages/turf-line-slice-along/types.ts

```diff
@@ -0,0 +1 @@
+import { lineSliceAlong } from "./index";
```

Comment:
Probably needs some usage if this file exists.

Reply:
Unless I'm missing your point, this is probably as good as we can make it, at least for a JS project. We could add example calls to lineSliceAlong, but all that's will confirm is we've written the example calls to match the signatures defined in index.d.ts

Alternative tack: do we not bother doing static type checking in our remaining JS packages? I don't think it's really adding much.

Reply:
Yeah if we aren't getting benefit from types.ts testing, maybe we can just remove it entirely. Something like https://api-extractor.com/ might be better for making sure we don't actually break APIs than these manual tests.

Reply:
Ok so I dug in on one of the types.ts files and it originally came from here which is part of the early Typescript implementation, so maybe in 2024 they are less relevant.

https://github.com/Turfjs/turf/commit/74140e862cc7ba44082353de824cb76e0024a9c1

Reply:
So, sounds like we're happy to ditch the types.ts for remaining JS projects, knowing that we'll have actually useful static type checking once they're converted to TS?

API extractor worth investigating too 👍

Reply:
I'd like to get more consensus around this decision, especially since I wasn't around when the decision was made to include them. I don't want to delay this PR indefinitely though.

Reply:
Sure thing. I'll proceed as if we will retire them for the remaining JS projects, and if justification comes up to keep them, it's easy to revert before we merge this PR.

Reply:
The above is done. To further simplify configuration (MRL mainly) all packages now run tsc. For the JS projects though it's pretty much a no-op as it can't make any guarantees about the type safety of the JS code.